recipe/meta.yaml

```diff
@@ -40,18 +38,29 @@ test:
 about:
   home: http://had.co.nz/reshape
   license: MIT
-  summary: 'Flexibly restructure and aggregate data using  just two functions: melt and cast.'
+  summary: 'Flexibly restructure and aggregate data using just two functions: melt and cast.'
   license_family: MIT
-
   license_file:
-    - {{ environ["PREFIX"] }}/lib/R/share/licenses/MIT
+    - '{{ environ["PREFIX"] }}/lib/R/share/licenses/MIT'
     - LICENSE
+
 extra:
   recipe-maintainers:
     - conda-forge/r
-    - johanneskoester
-    - bgruening
-    - daler
-    - jdblischak
-    - cbrueffer
-    - dbast
+
+# Package: reshape
+# Version: 0.8.8
+# Title: Flexibly Reshape Data
+# Description: Flexibly restructure and aggregate data using just two functions: melt and cast.
+# Authors@R: person("Hadley", "Wickham", , "hadley@rstudio.com", c("aut", "cre"))
+# URL: http://had.co.nz/reshape
+# Depends: R (>= 2.6.1)
+# Imports: plyr
+# License: MIT + file LICENSE
+# LazyData: true
+# NeedsCompilation: yes
+# Packaged: 2018-10-23 18:21:49 UTC; hadley
+# Author: Hadley Wickham [aut, cre]
+# Maintainer: Hadley Wickham <hadley@rstudio.com>
+# Repository: CRAN
+# Date/Publication: 2018-10-23 18:50:03 UTC
```

Comment:
The metadata can also be left off IMHO, it's not very useful and gets stale very quickly.

Reply:
We decided to keep the metadata. Please see https://github.com/bgruening/conda_r_skeleton_helper/issues/25 and https://github.com/bgruening/conda_r_skeleton_helper/pull/31. If you would like to propose removing the metadata, please open an Issue in the helper script repo to discuss. I don't feel strongly about it. I agree it gets stale quickly, but on the other hand it is convenient for a quick spot check of the recipe. When it's not there, I usually end up having to manually look up the `DESCRIPTION` file on CRAN